pkg/insights/insightsuploader/insightsuploader.go

```diff
@@ -223,7 +227,12 @@ func (c *Controller) Upload(ctx context.Context, s *insightsclient.Source) (stri
 	if err != nil {
 		return "", statusCode, err
 	}
+
 	klog.Infof("Uploaded report successfully in %s", time.Since(start))
+
+	// Update LastReportTime after successful upload
+	updateClusterOperatorLastReportTime(ctx, configClient)
```

Comment:
We should keep the error handling as it was in the `gather_commands.go`  and log the error returned from this function.